src/main/java/io/tus/java/client/TusInputStream.java

```diff
@@ -19,8 +19,8 @@
      *
      * @param stream The stream to read from
      */
-    public TusInputStream(InputStream stream) {
```

Comment:
Removed modifier here, hope that is okay for you

Reply:
That's a compatibility-breaking change, isn´t it? I would like to avoid those.